fissa/core.py

```diff
@@ -1260,8 +1260,19 @@ def run_fissa(
         The vector ``result[c, t][0, :]`` is the trace from cell ``c`` in
         trial ``t``. If ``return_deltaf=True``, this is Δf/f\ :sub:`0`;
         otherwise, it is the decontaminated signal scaled as per the raw
+        signal. f\ :sub:`0` is the baseline as calculated from the raw
         signal.
 
+    raw : 2d numpy.ndarray of 2d numpy.ndarrays of np.float64
+        The raw traces without separation.
+        The vector ``raw[c, t][0, :]`` is the ROI trace from cell ``c`` in
+        trial ``t``. The vector ``raw[c, t][i, :]`` for i>=1 the  trace from
+        cell ``c`` in trial ``t``, from neuropil region ``i-1``.
+        If ``return_deltaf=True``, this is Δf/f\ :sub:`0`; otherwise it's
+        the raw extracted signal.
+
+        .. versionadded:: 1.0.0
+
```

Comment:
You don't need to add versionadded to unreleased functions/methods. The version where `raw` is added is the same as the version where `run_fissa` is added.

```suggestion
```

Reply:
Argh, of course. Updated that.